src/payment/payment.controller.ts

```diff
@@ -17,18 +18,23 @@ export class PaymentController {
     return this.paymentService.findAll();
   }
 
+  @Get('users/:userId')
```

Comment:
i tutaj `userID`, bo wszędzie piszemy tak, że jeśi w camelCase jest akronim to jest z wielkich ;)